assets/scss/lib/partials/_cta.scss

```diff
@@ -0,0 +1,35 @@
+/**
+* @file
+* This file is an override of _cta.scss in localgov_theme.
+*/
```

Comment:
Minor issue.  A space is missing from the beginning of these lines which is breaking indentation.